pyproject.toml

```diff
@@ -0,0 +1,48 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+[project]
+name = "cuda-quantum"
+dynamic = ["version"]
+keywords = [ "CUDA Quantum", "Quantum Computing", "CUDA", "quantum" ]
+description="Python bindings for the CUDA Quantum toolkit for heterogeneous quantum-classical workflows"
+authors = [{name = "NVIDIA Corporation & Affiliates"}]
+readme = "README.md"
+requires-python = ">=3.8"
+license = { file="LICENSE" }
+
+[project.urls]
+Homepage = "https://developer.nvidia.com/cuda-quantum"
+Documentation = "https://nvidia.github.io/cuda-quantum"
+Repository = "https://github.com/NVIDIA/cuda-quantum"
+Releases = "https://nvidia.github.io/cuda-quantum/latest/releases.html"
+
+[project.optional-dependencies]
+chemistry = [ "scipy==1.10.1", "openfermionpyscf==0.5" ]
+
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[tool.scikit-build]
+wheel.packages = ["python/cudaq"]
+build-dir = "_skbuild"
+sdist.include = ["_version.py"]
+metadata.version.provider = "scikit_build_core.metadata.setuptools_scm"
+cmake.minimum-version = "3.26"
+cmake.build-type = "Release"
+cmake.verbose = true
+cmake.args = [ 
+    "-DCMAKE_COMPILE_WARNING_AS_ERROR=OFF", # FIXME
```

Comment:
Are there any new cmake warnings showing up here?

Reply:
A couple of commits back everything here build without a warning. Due to a recent change (not sure where and not sure if this PR or not) there is a warning generated when building nlopt when compiling with the -static flag only. I meant to investigate that warning (usually warnings are there for a reason). I suggest to do that as part of the next PR that sets up the wheel build workflows.